san/sanbase_graphql.py

```diff
@@ -640,3 +612,8 @@ def _format_to_date(datetime_obj_or_str):
     dt = iso8601.parse_date(datetime_obj_or_str) + \
         datetime.timedelta(hours=23, minutes=59, seconds=59)
     return dt.isoformat()
+
+def _format_return_fields(list_to_be_iterated):
```

Comment:
`list_to_be_iterated` can be renamed to `fields` cause there is no more need to distinguish the two params cause  now we have only one.